src/braket/device_schema/gate_model_parameters_v1.py

```diff
@@ -22,6 +22,8 @@ class GateModelParameters(BraketSchemaBase):
 
     Attributes:
         qubitCount: number of qubits for a device
+        noQubitRewiring: whether or not to run the circuit with the exact qubits chosen,
```

Comment:
"If `False`, no qubit rewiring is allowed. If `True`, qubit rewiring is allowed"?

Reply:
Will do (boolean values should be switched though)

Reply:
Oops, yeah. Actually that makes me wonder if the double negative is confusing - what do you think about just "qubitRewiring"?

Reply:
The negation in the name is indeed confusing. We should try to follow the standard naming patterns if possible (as we do for the APIs), since the SDK is the interface that the customers interact most with. Is there a standard for naming boolean fields ?

Reply:
The problem with that is that `True` basically has to become the default. Moreso, a corresponding `qubitRewiringSupported` property would be incorrect; we want to support the ability to _not_ rewire, since rewiring is the default.